src/Compute/Compute/help/New-AzVmssConfig.md

```diff
@@ -19,7 +19,7 @@ New-AzVmssConfig [[-Overprovision] <Boolean>] [[-Location] <String>] [[-Tag] <Ha
  [[-SkuTier] <String>] [[-SkuCapacity] <Int32>] [[-UpgradePolicyMode] <UpgradeMode>]
  [[-OsProfile] <VirtualMachineScaleSetOSProfile>] [[-StorageProfile] <VirtualMachineScaleSetStorageProfile>]
  [[-NetworkInterfaceConfiguration] <VirtualMachineScaleSetNetworkConfiguration[]>]
- [[-Extension] <VirtualMachineScaleSetExtension[]>] [-SkipExtensionsOnOverprovisionedVMs]
```

Comment:
Change a parameter input is a breaking change. 
As the src codes are included in the repo, we need a further discussion.

Reply:
@msJinLei  This PR does not change the input parameter.   I just re-generated the help file.  The existing help file is incorrect, because the type is PSVirtualMachineScaleSetExtension, not VirtualMachineScaleSetExtension.  So the line of help file is changed to the following line:

[[-Extension] <PSVirtualMachineScaleSetExtension[]>] [-SkipExtensionsOnOverprovisionedVMs]